src/main/java/teammates/ui/webapi/output/SessionResultsData.java

```diff
@@ -371,18 +437,24 @@ public CommentOutput getParticipantComment() {
         @Nullable
         private final String lastEditorName;
 
-        public CommentOutput(FeedbackResponseCommentAttributes frc, String commentGiverName, String lastEditorName) {
+        public CommentOutput(FeedbackResponseCommentAttributes frc,
+                             String commentGiver, String commentGiverName,
+                             String lastEditorEmail, String lastEditorName) {
             super(frc);
+            this.commentGiver = commentGiver;
+            this.commentGiverName = commentGiverName;
+            this.lastEditorEmail = lastEditorEmail;
+            this.lastEditorName = lastEditorName;
+        }
 
```

Comment:
So, decided to pass in `null` and remove if-else check? :)

Reply:
Applied builder pattern instead 👍

Reply:
Looks great!